src/test/unit/marker-data.test.js

```diff
@@ -584,6 +585,27 @@ describe('deriveMarkersFromRawMarkerTable', function() {
       category: 0,
     });
 
+    // Test for a marker with a stack
+    expect(markers[11]).toEqual({
+      data: {
+        // Stack property is converted to a cause.
+        cause: {
+          stack: 2,
+          tid: 1111,
+          // The cause's time hasn't been changed.
+          time: 1,
+        },
+        filename: '/foo/bar/',
+        operation: 'create/open',
+        source: 'PoisionOIInterposer',
+        type: 'FileIO',
+      },
+      start: 22,
+      end: 24,
+      name: 'FileIO',
+      category: 0,
+    });
```

Comment:
That's kinda hard to check when looking at the test, since the marker definitions are in the fixtures. But it's the case for all these tests. So I guess it's fine to keep. But it can be something to think about in the future.

Reply:
Yeah, I don't like these tests at all. Moreover with the IPC stuff the content and parent markers don't have the same order, this is painful...
Obviously this was out of scope for this patch but I'd like that we change that as well...